### DISCUSSION TITLE
Implementation of "break n"

### DISCUSSION THREAD

**Discussion by TheBlackPlague** [dotnet/csharplang#5593] [Language Ideas] [answered]
I am surprised that this doesn't already exist, but it should.

Case:
```cs
for () {
  for () {
    break 2;
  }
}
```
This would break through both loops and allow continuation. There exist suggestions to use "goto" for something like this, but that's an ugly way to do it when you can enable `break n`.

**Comment by pinkfloydx33** [comment, marked as answer]
I'm positive there are others, but found these with a quick search:

https://github.com/dotnet/csharplang/issues/869
https://github.com/dotnet/csharplang/issues/1597
https://github.com/dotnet/csharplang/issues/1907
https://github.com/dotnet/csharplang/issues/2024
https://github.com/dotnet/csharplang/issues/3511

**Reply by Joe4evr** [reply]
This one too: https://github.com/dotnet/roslyn/issues/5883

**Reply by TheBlackPlague** [reply]
I do agree. I'll continue this conversation on #869 .
